subsys/bluetooth/services/ots/otc.c

```diff
@@ -33,61 +37,13 @@
 /* TODO: KConfig options */
 #define OTS_CLIENT_INST_COUNT   1
 
-NET_BUF_POOL_FIXED_DEFINE(ots_c_read_queue, 1,
-			  sizeof(struct bt_gatt_read_params), 8, NULL);
-
-#define OTS_WORK(_w) CONTAINER_OF(_w, struct otc_instance_t, work)
-
-static int l2cap_accept(struct bt_conn *conn, struct bt_l2cap_chan **chan);
-static struct net_buf *l2cap_alloc_buf(struct bt_l2cap_chan *chan);
-static int l2cap_recv(struct bt_l2cap_chan *chan, struct net_buf *buf);
-static void l2cap_sent(struct bt_l2cap_chan *chan);
-static void l2cap_status(struct bt_l2cap_chan *chan, atomic_t *status);
-static void l2cap_connected(struct bt_l2cap_chan *chan);
-static void l2cap_disconnected(struct bt_l2cap_chan *chan);
-
-#define CREDITS         10
-#define DATA_MTU        (BT_OTC_MAX_WRITE_SIZE * CREDITS)
-
-NET_BUF_POOL_FIXED_DEFINE(otc_l2cap_pool, 1, DATA_MTU, 8, NULL);
-
-struct l2ch {
-	struct k_delayed_work recv_work;
-	struct bt_l2cap_le_chan ch;
-};
-
-#define L2CH_CHAN(_chan) CONTAINER_OF(_chan, struct l2ch, ch.chan)
-#define L2CH_WORK(_work) CONTAINER_OF(_work, struct l2ch, recv_work)
-#define L2CAP_CHAN(_chan) _chan->ch.chan
-
 struct dirlisting_record_t {
```

Comment:
Not sure.
I believe the `ots_dir_list_internal.h` contains what is used by the `ots_dirlist.c`, while this struct is used by the client.
It could make sense to move it there, but should then also the directory listing client code be moved there?